config/dev-config.yaml

```diff
@@ -0,0 +1,5 @@
+tasknews: 
```

Comment:
afaik in cloud environments the preferred way is ti use environment variables (please correct me if I'm wrong)

Reply:
fixed